.travis.yml

```diff
@@ -4,33 +4,47 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
     - CONDA_PY=27
-    - CONDA_PY=34
     - CONDA_PY=35
+    - CONDA_PY=36
```

Comment:
Wondering if there's already an automated way to get all of the feedstocks re-rendered with the latest conda-smithy?